src/main/resources/application.properties

```diff
@@ -9,5 +9,5 @@ spring.jpa.properties.hibernate.format_sql=true
 spring.jpa.properties.hibernate.dialect=MariaDB
 
 application.security.jwt.secret-key=404E635266556A586E3272357538782F413F4428472B4B6250645367566B5970
-application.security.jwt.expiration=86400000 # a day
-application.security.jwt.refresh-token.expiration=604800000 # 7 days
+application.security.jwt.expiration=86400000
+application.security.jwt.refresh-token.expiration=604800000
```

Comment:
Why did you delete these comments? They made perfect sense and clearly outlined what these "magic numbers" mean